modules/statistics/php/stats_behavioural.class.inc

```diff
@@ -160,19 +162,43 @@ class Stats_Behavioural extends \NDB_Form
             $vl     = $row['VLabel'];
             $c      =& $this->tpl_data['behaviour']['C' . $center];
 
-            $c[$vl]['total']   += $row['val'];
-            $c['all']['total'] += $row['val'];
+            if (isset($c[$vl]['total'])) {
+                $c[$vl]['total'] += $row['val'];
+            } else {
+                $c[$vl]['total'] = $row['val'];
+            }
+            if (isset($c['all']['total'])) {
+                $c['all']['total'] += $row['val'];
+            } else {
+                $c['all']['total'] = $row['val'];
+            }
             if ($row['Data_Entry'] == 'Complete') {
-                $c[$vl]['complete']   += $row['val'];
-                $c['all']['complete'] += $row['val'];
+                if (isset($c[$vl]['complete'])) {
+                    $c[$vl]['complete'] += $row['val'];
+                } else {
```

Comment:
IMHO would have been clearer with:
$c[param]['param2'] = $c[param]['param2'] ?? 0;
$c[param]['param2'] += $row['val'];   // or other math

Reply:
I thought of suggesting that, but I think the `?? 0` will only suppress the undefined reference errors on the right hand side of the expression where the coalesce is used, there's still a problem with the right hand where it's 2 levels deep.

Reply:
Yes, of course, the first level need to exist. I made that assumption.